pre-docs/bolt_installing.md

```diff
@@ -194,6 +194,14 @@ The Puppet repository for the YUM package management system is [http://yum.puppe
         sudo yum install puppet-bolt
         ```
 
+    -   Enterprise Linux 8
+
+        ```
+        sudo rpm -Uvh https://yum.puppet.com/puppet6/puppet6-release-el-8.noarch.rpm
+        sudo yum install puppet-bolt
```

Comment:
Should we use `dnf` instead of `yum`? They're both symlinks to the same thing.

Reply:
For all el examples? Or just 8?

Reply:
Only for 8

Reply:
It looks like maybe it actually is `yum` on EL 8. There's apparently been some convergence (yum 4 is based on dnf??). It's quite unclear to me which one is the "right" command, so leaving it this way seems fine.